tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs

```diff
@@ -56,5 +58,29 @@ public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags
             // or we couldn't populate it completely (some tags is better than no tags)
             return scope;
         }
+
+        public static string GetQueueName(string queueUrl)
+        {
+            if (queueUrl is null)
+            {
+                return queueUrl;
+            }
+
+            var lastSeparationIndex = queueUrl.LastIndexOf('/') + 1;
+            return queueUrl.Substring(lastSeparationIndex);
+        }
+
+        internal static string GetOperationName(Tracer tracer, string spanKind) => tracer.CurrentTraceSettings.Schema.Version switch
```

Comment:
More code style nitpicking, but having two methods `GetOperationName()` and `GetOperationNameHelper()` seems a little convoluted just to use `switch` expressions instead of a series of `if` statements. The names also don't help. Why not use a single method instead?

```csharp
internal static string GetOperationName(Tracer tracer, string spanKind)
{
    if (tracer.CurrentTraceSettings.Schema.Version == SchemaVersion.V0)
    {
        return SqsRequestOperationName;
    }

    if (spanKind == SpanKinds.Consumer)
    {
        return tracer.CurrentTraceSettings.Schema.Messaging.GetInboundOperationName(SnsOperationName);
    }

    if (spanKind == SpanKinds.Producer)
    {
        return tracer.CurrentTraceSettings.Schema.Messaging.GetOutboundOperationName(SnsOperationName);
    }

    return $"{SnsOperationName}.request";
}
```

Reply:
Thanks Lucas, I'm gonna use your approach, I really didn't like having two different methods for the same purpose. But I didn't think about this!

This is cleaner and still easier to read!

Reply:
Slightly edited it to use a switch at the end, which is recommended.